packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -556,13 +556,59 @@ void main() {
 
     await tester.pumpWidget(build(controller));
 
-    expect(tester.getTopLeft(find.text('Alabama')), const Offset(-100.0, 0.0));
-    expect(tester.getBottomRight(find.text('Alabama')), const Offset(900.0, 600.0));
+    expect(tester.getTopLeft(find.text('Alabama')), Offset.zero);
```

Comment:
why this change?

Reply:
it looks like we changed the behavior when viewportFraction > 1
The page start at the very left edge of the page instead of after extra space.

Reply:
Yeah it does change the behavior when `viewportFraction > 1` and breaks the API (although I doubt it would really affect anyone, and starting from 0 makes more sense to me) . Context: https://github.com/flutter/flutter/pull/8539

Reply:
What's the motivation for changing this?

Please add a description of the changed behavior to the PR description.

Reply:
I actually think the original behavior make more sense to me. I would expect the viewport is center align with the page, that is why you have the padding when viewportfraction<1. If the same thing apply when viewportfraction>1, the original behavior is expected. and i am not quite sure how https://github.com/flutter/flutter/pull/8539 related to this behavior. Can you elaborate?

Reply:
Ah I thought a weird `viewportFraction` like 1.3 would mess up the snapping behavior but that's not true. Thanks for pointing out. But `SliverPadding`s can't go negative. I'll find an alternative for it.